user_guide/intro_nav_popup.rst

```diff
@@ -0,0 +1,50 @@
+.. _user-guide-intro-popups:
+
+Flash Notifications and Dialog Windows
+======================================
+
+In the course of the system usage, there sometimes appears the need to inform the user about some events in the system,
+for example some operation has been executed or has failed, or you need to be reminded about a task or event.
+
+Success flash messages are displayed against a green background:
+
+|
+
+.. image:: ./img/navigation/popup/flash_success.png
+
+|
+
+If something has gone wrong, the flash message is displayed against a red background, to attract  your attention and 
+make sure you have noticed it. 
+
+|
+
+.. image:: ./img/navigation/popup/flash_error.png
+
+|
+
+Other flash messages have a yellow background. 
+
+|
+
+.. image:: ./img/navigation/popup/flash_task.png
+
+|
+
+Sometimes, the flash messages can contain some links to other pages, such as the task reminder above, or this 
+synchronization message:
+
+|
+
+.. image:: ./img/navigation/popup/flash_link.png
+
+|
+
+Flash messages appear on the top of the screen for several seconds and disappear.
+
+If user decision is obligatory, and the system cannot go on without it, there will appear a dialog window. The 
+dialog windows have buttons and will not disappear until the decision has been taken:
```

Comment:
Edit: If the user must make a decision before the system can go on, a dialog window will appear. The dialog windows have buttons and will not disappear until a decision has been made: